### PR TITLE
Homework week2

### PR DESCRIPTION
**Pull request by gulyener**

### CODE REVIEW

**Comment by nudge** [comment]
It looks like you created your `homework-week2` branch on top of your `homework-week1` branch, therefore week1 homework appears in this pull request. Next time, don't forget to `git checkout master` first, then `git checkout -b homework-week_x` to create a new branch based on top of `master`.

File: Week2/homework/maartjes-work.js [hunk 1]
```diff
@@ -42,20 +42,72 @@ const tuesday = [
   },
 ];
 
-const maartjesTasks = monday.concat(tuesday);
+// I changed maartjesTasks variable into let in this exercise because I wanted to make changes in the original array.
+// I know that is not good practice but since at the end maartjesTasks and maartjesHourlyRate is used, I thought I should go from that.
+
+let maartjesTasks = monday.concat(tuesday);
```
Comment (nudge) [inline Week2/homework/maartjes-work.js]: The other way not to modify `maartjesTasks` is to put your code (line 77-90)  into the `computeEarnings()` function. I would suggest this as it keeps all the earnings calculations code into one function.

File: Week2/homework/maartjes-work.js [hunk 2]
```diff
@@ -42,20 +42,72 @@ const tuesday = [
...
+  duration: task.duration / 60,
+}));
+
+console.table(maartjesTasks);
+
+// Filter out every task that took less than two hours (remove from the collection)
+
+console.log('\nTasks that take more than two hours: \n');
+maartjesTasks = maartjesTasks.filter(task => task.duration >= 2);
+console.table(maartjesTasks);
```
Comment (nudge) [inline Week2/homework/maartjes-work.js]: Using `console.table` is nice!

File: Week2/homework/map-filter.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  // First we filter out the even numbers and then map the odd numbers
+  return numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```
Comment (nudge) [inline Week2/homework/map-filter.js]: Great!

**Comment by nudge** [comment]
Overall great use of `map`, `filter` and `reduce`, nice work!

File: Week2/homework/maartjes-work.js (hunk 2 above)
Comment (gulyener) [reply Week2/homework/maartjes-work.js]: Thanks I wasn't sure about it

File: Week2/homework/maartjes-work.js (hunk 1 above)
Comment (gulyener) [reply Week2/homework/maartjes-work.js]: I applied this change now and it feels and looks better thank you!

**Pull request closed without merging**